src/main/webapp/resources/js/components/menu/EllipsisMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Dropdown, Menu } from "antd";
+import { MoreOutlined } from "@ant-design/icons";
+
+export interface EllipsisMenuProps {
```

Comment:
You don't need to create this, just import `DropDownProps` from `antd`.  In your argument list change `menu` to `overlay`, make sure you update to spread any other passed in props since there are a couple other on the dropdown that might be used at some other point.

Reply:
Updated in  288f7a6dcd3e02b97ceb3a5535ab6bb1fd0d5e70